go.mod

```diff
@@ -33,8 +33,8 @@ replace github.com/prometheus/prometheus => github.com/prometheus/prometheus v0.
 require (
 	cloud.google.com/go/storage v1.30.1 // @grafana/backend-platform
 	cuelang.org/go v0.6.0-0.dev // @grafana/grafana-as-code
-	github.com/Azure/azure-sdk-for-go v65.0.0+incompatible // @grafana/backend-platform
-	github.com/Azure/go-autorest/autorest v0.11.28 // @grafana/backend-platform
+	github.com/Azure/azure-sdk-for-go v65.0.0+incompatible // @grafana-partner-datasources
+	github.com/Azure/go-autorest/autorest v0.11.28 // @grafana-partner-datasources
```

Comment:
This one doesn't look like it belongs to us, it's referenced in `pkg/extensions`